container-host-files/wid/README.md

```diff
@@ -0,0 +1,15 @@
+# SCF workspace in Docker
+
+You can build your own SCF workspace docker image by using:
+
+`
+    docker build -t your_org/wid:1.0.0 .
+`
+<br>
+<br>
+
+**NOTICE** <br>
+1. You need to run Docker container with **--privileged** parameter to allow start Docker deamon in the Docker container and use volume parameter for inner docker data folder, such as:
+`docker run -it --privileged --volume “/root/scf:/scf” --volume “/root/inner_docker_volume:/var/lib/docker” --name test your_org/wid:1.0.0 /bin/bash`
```

Comment:
From trial and error: Just exposing the docker socket directly (`--volume /var/run/docker.sock:/var/run/docker.sock`) will _not_ work, because the volume mounts will get all confused.  Unless you mount with `-v $PWD:$PWD` instead of `/scf`, then thing will map correctly.

Either way, running as root makes it annoying (unless you're dealing with disposable checkouts like concourse is).

Reply:
Sorry, I didn't understand it well. What is the error?
--volume “/root/scf:/scf”, these are the folder on your machine: docker container folder.
And `docker socket` is?

Reply:
I was attempting to not need to start a separate docker inside the container (instead, using the host docker).  This gets rid of the need for a `/var/lib/docker` mount.  Hence mounting `/var/run/docker.sock` into the container too.  But that's more me playing with the thing than something we actually need.